### PR TITLE
Bug 1865329 - Fix secret debug info text theme

### PR DESCRIPTION
**Pull request by rahulsainani**
Fixes text theme for secret debug info screen.

### Pull Request checklist

- [ ] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/firefox-android/blob/main/docs/changelog.md) or does not need one
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/firefox-android/blob/main/docs/shared/android/accessibility_guide.md) or does not include any user facing features

### After merge
- **Breaking Changes**: If this is a breaking Android Components change, please push a draft PR on [Reference Browser](https://github.com/mozilla-mobile/reference-browser) to address the breaking issues.

### To download an APK when reviewing a PR (after all CI tasks finished running):
1. Click on `Checks` at the top of the PR page.
2. Click on the `firefoxci-taskcluster` group on the left to expand all tasks.
3. Click on the `build-apk-{fenix,focus,klar}-debug` task you're interested in.
4. Click on `View task in Taskcluster` in the new `DETAILS` section.
5. The APK links should be on the right side of the screen, named for each CPU architecture.

### GitHub Automation
https://bugzilla.mozilla.org/show_bug.cgi?id=1865329

### CODE REVIEW

**Review by MozillaNoah — APPROVED** [review]

File: fenix/app/src/main/java/org/mozilla/fenix/settings/SecretDebugSettingsFragment.kt [hunk 1]
```diff
@@ -39,41 +42,46 @@ class SecretDebugSettingsFragment : Fragment() {
...
                 }
             }
         }
     }
 }
 
 @Composable
-private fun DebugInfo() {
-    val store = components.core.store
+private fun DebugInfo(browserStore: BrowserStore) {
```
Comment (gabrielluong) [inline fenix/app/src/main/java/org/mozilla/fenix/settings/SecretDebugSettingsFragment.kt:53]: I don't think we should ever pass `BrowserStore` as a parameter to our Composables. Could we observe the `regionState` above and pass this state instead?

Comment (rahulsainani) [reply fenix/app/src/main/java/org/mozilla/fenix/settings/SecretDebugSettingsFragment.kt]: We can certainly, extracted a top level composable that just observes the state and passes the `regionState` to `DebugInfo()`

**Review by gabrielluong — APPROVED** [review]

File: fenix/app/src/main/java/org/mozilla/fenix/settings/SecretDebugSettingsFragment.kt [hunk 2]
```diff
@@ -39,41 +41,49 @@ class SecretDebugSettingsFragment : Fragment() {
...
                 }
             }
         }
     }
 }
 
 @Composable
-private fun DebugInfo() {
-    val store = components.core.store
+private fun SecretDebugSettingsScreen() {
```
[the remark this answers is not in the document]
Comment (rahulsainani) [reply fenix/app/src/main/java/org/mozilla/fenix/settings/SecretDebugSettingsFragment.kt:52]: hmm good question, I think it's semantically correct, the lower level nodes can be reused in any other top level Screens/Sheets/etc nodes. The [docs](https://developer.android.com/jetpack/compose/mental-model) also mention the same idea

[the remark this answers is not in the document]
Comment (gabrielluong) [reply fenix/app/src/main/java/org/mozilla/fenix/settings/SecretDebugSettingsFragment.kt:52]: I also confirmed it for myself by searching for `Screen` in the Compose Sample Apps - [Example](https://github.com/android/compose-samples/blob/2feda1a156e33be82e70cc37e479e96eda3cdf5f/Jetsurvey/app/src/main/java/com/example/compose/jetsurvey/signinsignup/SignInScreen.kt#L56). I think we are gucci.

**Pull request merged**